packages/@webex/plugin-meetings/src/common/browser-detection.ts

```diff
@@ -1,5 +1,7 @@
+// @ts-nocheck
+
 import bowser from 'bowser';
-import {memoize} from 'lodash';
+import { memoize } from 'lodash';
```

Comment:
there is still a lot of pure formatting changes in this PR not related to adding types, this is one of the examples

Reply:
I missed to revert the formatting from couple of files. Have updated them now. Some files may still have very minor formatting which shouldn't cause any issue but if we want to revert all formatting strictly, please let me know, will remove them as well.